### PR TITLE
[Intel MKL] Enabling QuantizeV2 with native format

### PR DESCRIPTION
**Pull request by mahmoud-abuzaina**

### CODE REVIEW

**Review by penpornk — COMMENTED** [review]
Thank you for the PR and I'm so sorry for the delay!
This PR looks good to me. I'll approve it after #45107 is merged.

**Review by penpornk — APPROVED** [review]
Approving this now since it doesn't depend on #45107 anymore.

**Review by gbaned — APPROVED** [review]

**Pull request merged**
